docs/user-guide/src/contributing/services_contacts.md

```diff
@@ -0,0 +1,118 @@
+- tide: Handles automatic merging of pull requests.
+
+Prow is configured through files in [project-infra](https://github.com/metal3-io/project-infra/tree/main/prow/config).
+
+#### Mino
+
+[Minio](https://min.io/) is used for storing the logs produced by Prow when running tests.
+It stores the data in an Azure bucket.
+
+#### Ingress Nginx
+
+[Ingress Nginx](https://kubernetes.github.io/ingress-nginx/deploy/) is used as ingress controller in the cluster.
+It is through this we expose the public endpoints, e.g. the Prow dashboard.
+
+#### Cert-manager
+
+[Cert-manager](https://cert-manager.io/) is used for certificate management in the cluster.
+This is mainly for getting Let's Encrypt certificates for the Ingresses.
+
+### Prow Kubernetes Cluster
+
+The Kubernetes cluster where Prow is running.
+
+### IDS Wazuh
+
+[Wazuh](https://wazuh.com/) is a "security platform" used to keep the bare metal lab secure.
+See [Nordix wiki](https://wiki.nordix.org/pages/viewpage.action?pageId=47546865).
+
+### Jenkins
+
+The [Nordix Jenkins instance](https://jenkins.nordix.org/) is used for all "heavier" tests and build jobs (e.g. integration tests, e2e tests and node image building jobs).
+It is managed by the Nordix admins, but we provide the Jenkins agents, pipelines and configure secrets.
+
+The jobs are defined using [Jenkins Job Builder](https://jenkins-job-builder.readthedocs.io/en/latest/) and are stored in the [cicd repo on the Nordix Gerrit](https://gerrit.nordix.org/admin/repos/infra/cicd,general).
+Each job references a pipeline in [project-infra](https://github.com/metal3-io/project-infra/tree/main/jenkins/jobs) or [metal3-dev-tools](https://github.com/Nordix/metal3-dev-tools/tree/master/ci/jobs) that is executed by Jenkins when the job is triggered.
+
+### Jump hosts
+
+We have jump hosts in Cleura which are used to avoid having to expose every developer VM to the internet.
+See [this wiki page](https://wiki.nordix.org/display/IN/Jumphost+Setup).
+
+### Bare metal lab
+
+We have access to a server rack with actual physical servers.
+This is used for some CI jobs as well as for manual testing.
+We are responsible for the software on these machines but not for the hardware itself.
+More details can be found on the [BML wiki](https://wiki.nordix.org/display/CPI/Bare+Metal+Lab).
+
+## Contact information to those we rely on
+
+When things break, it is often because of issues in the underlying infrastructure.
+In these cases we need to know who to contact to report the issues and where to look for known issues (e.g. status pages).
+
+### Cleura
+
+Contanct: <support@cleura.com>
+
+Status page: <https://www.cnstatus.com/>
+
+Cleura (former CityCloud) is the cloud platform we run most of our things on.
+All the Jenkins jobs, Prow and all developer VMs are here.
+They basically donate the resources to us and the normal support email is available if we need to reach them.
+
+### Nordix services
+
+Contact: <discuss@lists.nordix.org>
+
```

Comment:
Does nordix admins respond to requests made from someone outside of EST? Do we expect any community member to create request there?

Reply:
Good point. I'm not sure how much we should expect the community to engage but it would be nice to prepare for the possibility that someone outside of EST/Nordix gets involved.

Anyway, the most important part for me is to have this documented somewhere. If that somewhere should be accessible to the community or just EST doesn't matter much to me. I see two main options:

1. [this PR] Make everything accessible to the community and figure out how people outside of EST can get involved. 
2. Make this doc internal to EST (for example on the Nordix wiki).

Let's discuss when people are back from vacation.